backends/openvino/tests/README.md

```diff
@@ -0,0 +1,67 @@
+# Unit Tests for OpenVINO Backend
+
+## Directory Structure
+
+Below is the layout of the `backends/openvino/tests` directory, which includes the necessary files for the example applications:
+
+```
+backends/openvino/tests
+├── ops                                 # Directory with base op test script and individual op tests.
+    ├── base_openvino_op_test.py        # Script which contains the base class for all op tests.
+    └── test_<op_name>.py               # Individual op tests scripts.
+├── models                              # Directory with model test scripts.
+    └── test_classification.py          # Test script for classification models.
+├── README.md                           # Documentation for unit tests (this file)
+└── test_runner.py                      # Script to execute unit tests.
+```
+
+## Executing Unit Tests
+
+### Prerequisites
+
+Before you begin, refer to instructions provided in [OpenVINO Backend for ExecuTorch](../README.md) to install openvino and setup executorch environment.
+Once openvino is installed and executorch environment is set, refer to [OpenVINO Backend Examples](../../../examples/openvino/README.md) to build openvino_example_runner.
+
+### Usage
+
+`test_runner.py` allows to run op or model tests for openvino backend.
+
+### **Arguments**
+- **`--build_folder`** (required):  
```

Comment:
Is this needed because you need to load library? I would consider making this part of pip package installed in site-packages.

Reply:
Updated tests to use python bindings. With the new changes, we do not need the `--build_folder` argument anymore. This argument is removed.